elliottlib/cli/find_builds_cli.py

```diff
@@ -162,7 +161,19 @@ def _fetch_builds_by_id(builds, product_version, session):
 def _fetch_builds_from_diff(from_payload, to_payload, product_version, session):
     green_print('Fetching changed images between payloads...')
     changed_builds = elliottlib.openshiftclient.get_build_list(from_payload, to_payload)
-    return [elliottlib.brew.get_brew_build(b, product_version, session=session) for b in changed_builds]
+    brew_session = koji.ClientSession(constants.BREW_HUB)
+    builds = []
+    for cb in changed_builds:
+        for tag in brew_session.listTags(cb):
+            name = tag.get('name')
+            if name:
+                break
+        if name:
```

Comment:
How about setting `name = None` first above? If there were no tags, name would not be defined. That may be impossible but ... I just get nervous seeing uninitialized things.